spec/features/admin/dashboard/index_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+RSpec.describe "Admin Dashboard Index", type: :feature do
+  before(:each) do
+    # merchant
+    @merchant_1 = create(:merchant)
+    
+    # customers
+    @customer_1 = create(:customer)
```

Comment:
This way of testing makes so much more sense compared to what we have been doing